build.gradle

```diff
@@ -343,13 +343,13 @@ allprojects {
   }
 
   plugins.withType(JavaBasePlugin) {
-    eclipse.classpath.defaultOutputDir = file('build-eclipse')
```

Comment:
Eclipse actually didn't like how `build-eclipse` was both the default and contained the folders from below. I shoved the "default" folder to the side.

Reply:
What _is_ the actual default? Is there a reason to override this at all?

Reply:
I'll check. Probably `bin` or some garbage like that. It might not matter
to override because we set the build for everywhere.

Reply:
Its `bin/default`.